src/renderer/widgets/CreateWallet/ui/MultisigWallet/components/WalletForm.tsx

```diff
@@ -80,7 +79,14 @@ export const WalletForm = ({ signatories, onContinue, isActive, isLoading, onGoB
     );
   });
 
-  const accountAlreadyExists = accounts.some((a) => a.accountId === multisigAccountId);
+  const accountAlreadyExists = wallets.some((wallet) => {
+    return (
+      !walletUtils.isWatchOnly(wallet) &&
+      accounts.some((account) => {
+        return account.accountId === multisigAccountId && account.walletId === (wallet as Wallet).id;
```

Comment:
this `wallet as Wallet` is very weird to me. For some reason, I got a TS error here saying that "Property 'id' does not exist on type 'never'". I really don't understand. Let me know if you have an idea, otherwise I'll research more to remove this type casting.